games-emulation/pcsxr/pcsxr-1.9.95.ebuild

```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
```

Comment:
Please cleanup old in a separate commit.

Reply:
Also, you didn't change EGIT_COMMIT and the Manifest didn't change, so I bet this is actually the same version ;)

We should change the ebuild to do something like:
if [[ ${PV} == *_p* ]] ; then
	EGIT_COMMIT=""
	SRC_URI="https://github.com/iCatButler/pcsxr/archive/${EGIT_COMMIT}.tar.gz -> ${MY_P}.tar.gz"
else
	SRC_URI="" # (fill this in)
fi